api_tests/src/shared.ts

```diff
@@ -198,7 +205,7 @@ export async function setupLogins() {
     // only needed the first time so do in this try
     await delay(10_000);
   } catch {
-    console.log("Communities already exist");
+    //console.log("Communities already exist");
```

Comment:
These comments were previously in the code, but were quite noisy during testing so I commented them out. I would still leave them in case the logging is needed to debug a test failure.